### PR TITLE
Add a 'Full' workspace

### LINKED ISSUE #8605: [MU4 Issue] Font face dropbox
Dropbox placement issues over the box. For example, look how Fira Sans is being placed from button to dropdown menu. The already-selected option menu should be placed over. There is also some issue when it comes to being stuck on the bottom of the preferences window.

https://www.dropbox.com/s/nxhh3ckd5qj6yhq/Screen%20Recording%202021-07-13%20at%2014.24.17.mov?dl=0

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**
generated on the fly, so same content as the master palette

### CODE REVIEW

File: mscore/menus.cpp [hunk 1]
```diff
@@ -1464,6 +1464,7 @@ void MuseScore::setDefaultPalette()
       paletteBox->addPalette(newFingeringPalette());
       paletteBox->addPalette(newRepeatsPalette());
       paletteBox->addPalette(newFretboardDiagramPalette());
+      paletteBox->addPalette(newAccordionPalette());
```
Comment (dmitrio95) [inline mscore/menus.cpp]: Isn't it necessary also to add it to workspace files to actually add it to palettes? Or you didn't intend to do so?

Comment (Jojo-Schmitz) [reply mscore/menus.cpp]: Actually I deliberatly didn't add it to any workspace, yet. And I'd rather remove the bagpipe embelshiments from the Advanced worksspace too.
And to be honest: I have no idea what that `setDefaultPalette()` does or is good for...

Comment (Jojo-Schmitz) [reply mscore/menus.cpp]: What do you (plural) think, should we also remove the bagpipe emblishments from the advanced palette? That and this accordion palette are really seldom used, usefull for those that need them, but too much for the general users, even for advanced ones IMHO. And those uses that need them on a regular basis a) still/then have them available in the master plattetes and b) can put them into their custom workspace.

Comment (Jojo-Schmitz) [reply mscore/menus.cpp]: As agreen on in the developers' chat I've now removed the bagpipe embellishments from the Advanced workspace

**Comment by anatoly-os** [comment]
I checked the PR. It looks good, but not for 3.1 release. Let's postpone a decision on this. I wonder how to use the new "Full" workspace semantic.

**Comment by anatoly-os** [comment]
Also, I would remove the bagpipe only if we provided the bagpipe extension with the workspace for bagpipes.

**Comment by Jojo-Schmitz** [comment]
? It is in the Full workspace

**Comment by anatoly-os** [comment]
I missed the new workspace file, lol. Ok, so you want to add one more workspace which changes the semantic of the Advanced workspace. I would go deeper and understand better which buttons could be removed from the toolbars (like 3 dots) in the Advanced workspace.

**Comment by Jojo-Schmitz** [comment]
It is not a new workspace __file__, but created in the fly from all palettes

**Comment by Jojo-Schmitz** [comment]
And progress on this? Seems to be a good addition to 3.3, isn't it?

File: mscore/workspace.cpp [hunk 2]
```diff
@@ -1013,6 +1014,13 @@ QList<Workspace*>& Workspace::workspaces()
                               }
                         }
                   }
+
+            Workspace* p = new Workspace;
+            p->setName("Full");
+            p->setTranslatableName("Full");
```
Comment (dmitrio95) [inline mscore/workspace.cpp]: It would probably be better to reuse `fullWorkspaceTranslatableName` variable here

Comment (Jojo-Schmitz) [reply mscore/workspace.cpp]: Of course. I guess `p->setTranslatableName(QString(fullWorkspaceTranslatableName));` should do the trick

Comment (Jojo-Schmitz) [reply mscore/workspace.cpp]: What about the name, "Full" or better "Complete"? The latter muíght be aeasier for translators, e.g. in Geman "Full" could mean "Voll" and "Vollständig", whereas "Complete" means "Vollständig" or "Komplett", a better match to the meaning?

Comment (Jojo-Schmitz) [reply mscore/workspace.cpp]: Maybe also renaming the master palette to the full or complete palette?

**Comment by Jojo-Schmitz** [comment]
BTW, it is not a suggestion really, much more of a bug, that accordion palette exists in the code since long (and pestering translators with string they can' see and test), it just has been forgotten to get exposed to the users, whereas the bagpipe palette should never have made it into the advanced workspace IMHO.

File: mscore/workspace.cpp [hunk 3]
```diff
@@ -40,6 +40,7 @@ QList<Workspace*> Workspace::_workspaces {};
 QList<QPair<QAction*, QString>> Workspace::actionToStringList {};
 QList<QPair<QMenu*  , QString>> Workspace::menuToStringList   {};
 
+const char* Workspace::fullWorkspaceTranslatableName{ QT_TRANSLATE_NOOP("Ms::Workspace", "Full") };
```
Comment (anatoly-os) [inline mscore/workspace.cpp]: @Jojo-Schmitz could you please remove the renaming from this PR? I'm ready to merge the changes related to Accordion palette and removing the Bagpipe embellishment, but renaming workspaces is something that needs additional discussion.

**Comment by anatoly-os** [comment]
@Jojo-Schmitz I suggest keeping the changes that add accordion palette and remove the Bagpipe Embellishment palette from the Advanced palette. All other changes need additional discussions and could be postponed.

File: mscore/workspace.cpp (hunk 3 above)
Comment (Jojo-Schmitz) [reply mscore/workspace.cpp]: There is no **renaming** workspaces, this 2nd commit is **adding** a 3rd workspace, generated on the fly.

**Comment by Jojo-Schmitz** [comment]
You mean to merge the 1st commit but postpone the 2nd, the addition a full workspace? I'm fine with that.

**Comment by anatoly-os** [comment]
Yes, exactly. Fine, I'll cherry-pick the first commit manually then.

**Comment by anatoly-os** [comment]
Done

**Comment by Jojo-Schmitz** [comment]
Why closed then? Can you reopen, so I can rebase with that 2nd part?

**Comment by Jojo-Schmitz** [comment]
No longer Bugfix (that part had been cherry picked), more Improvement and/or Suggestion

**Comment by Jojo-Schmitz** [comment]
Rebase would be needed, but a) I can't get that to work and b) this PR doesn't seem needed anymore anyway

**Pull request closed without merging**